apps/web/lib/components/kanban-card.tsx

```diff
@@ -21,23 +29,23 @@ function getStyle(provided: DraggableProvided, style: any) {
 	};
 }
 
-function setCommentIconColor(commentType: 'tagged' | 'untagged') {
-	let style;
+// function setCommentIconColor(commentType: 'tagged' | 'untagged') {
```

Comment:
@Anishali2 why this code commented out? Why we had it you think, how you replaced it?

Reply:
Let me check this